.github/workflows/update-fast-sync.yml

```diff
@@ -18,8 +18,8 @@ jobs:
         uses: actions/checkout@v4
       - name: Install dependencies
         run: |
-          pip3 install setuptools
-          pip3 install emoji
+          pip3 install setuptools --break-system-packages
+          pip3 install emoji --break-system-packages
```

Comment:
Where this comes from?

Reply:
@LukaszRozmej 
https://github.com/NethermindEth/nethermind/pull/7612

When you trigger this action form different branch, changes form branch are automatically included for some reason

Reply:
Action stopped working because of some changes in github runners (not only here but in other repos as well)

Reply:
feature/fast-sync-config-6-1